presto-parser/src/main/java/com/facebook/presto/sql/tree/ExplainAnalyze.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class ExplainAnalyze
```

Comment:
I thought of just adding it as an option, but I don't think the current "logical" and "distributed" options will apply. Explain analyze needs to render distributed, otherwise you can't include the CPU cost (which is collected at the stage level), so as an option it would then implicitly set `TYPE DISTRIBUTED`.

If you think it's better to make it an option Explain, I certainly do that though.